target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -232,6 +232,9 @@ ramips_board_detect() {
 	*"HC5861")
 		name="hc5861"
 		;;
+	*"HC5861B")
```

Comment:
Please drop this block and use the board compatible string instead as userspace boardname (`hiwifi,hc5861b`)